person.rb

```diff
@@ -0,0 +1,22 @@
+class Person
+  attr_accessor :id, :name, :age
+
```

Comment:
- [ ] [REQUIRED] Good job so far! The `:id` should only be able to be retrieved so only the Getter is defined, please replace the `:id` to `attr_reader` 🙏🏼 

```ruby
  attr_accessor :name, :age 
  attr_reader :id 
```